content/specs/versions3.md

```diff
@@ -0,0 +1,148 @@
+---
+description: A list of tools available out-of-the-box on Codemagic build machines.
+title: macOS build machine specification (Xcode 12.5+)
+aliases:
+  - '../releases-and-versions/versions3'
+weight: 1
+---
+
+## Hardware
+
+- Standard VM on Mac mini `2.3GHz Quad Core / 8GB`
+- Premium VM on Mac Pro `3.7GHz Quad Core / 32GB`
+
+## System
+
+- System version `macOS 11.2 (20D64)`
+- Kernel version `Darwin 20.3.0`
+- Disk `322GB (Free Space: 94GB)`
+
+## Pre-installed tools
+
+- Android tools `/usr/local/share/android-sdk`
+- aws `2.1.29`
+- cocoapods `1.10.1`
+- cordova `10.0.0`
+- curl
+- dart `2.10.5`
+- docker `20.10.2`
+- fastlane `2.176.0`
+- firebase `9.3.0`
+- flutter `2.0.1 ($HOME/programs/flutter)`
+- gem `3.1.4`
+- git `2.30.1`
+- gradle `6.1.1`
+- homebrew `3.0.4`
+- ionic `6.12.4`
+- jq
+- node `14.15.5`
+- npm `7.5.3`
+- python `3.8.7`
+- python3 `3.8.7`
+- ruby `2.7.2p137`
+- ssh
+- sudo
+- tar
+- unzip
+- wget
+- yarn `1.22.10`
+- yq
+- zip
+
+## Xcode 12.5 (12E5244e)
+
+This is the Xcode version used by default when you select `latest` in build settings. Other available versions are listed [here](#other-xcode-versions).
```

Comment:
Once we make this version available, it should be the edge, right? So I would change here "when you select 'edge'"